CRM/Utils/System/Drupal.php

```diff
@@ -853,10 +853,12 @@ public function synchronizeUsers() {
    * Similar to drupal_exit().
    */
   public function onCiviExit() {
-    if (!defined('MAINTENANCE_MODE') || MAINTENANCE_MODE != 'update') {
-      module_invoke_all('exit');
+    if (function_exists('drupal_session_commit')) {
```

Comment:
is it the case that module_invoke_all would also not exist? Just wondering why we skip both parts of this if drupal_session_commit does not exist

Reply:
ie. if feels like separately checking for module_invoke_all makes sense or only checking for module_invoke_all as the first one seems slightly better than just checking for drupal_session_commit - as it stands I feel it needs at least a comment to explain why we skip module_invoke_all if drupal_session_commit doesn't exist

Reply:
The logged error was related to `module_invoke_all` function itself. My thought was to check either of the function just to ensure CMS is bootstrapped which will avoid error to be recorded. Updated the above line to check for `module_invoke_all` function for readability 👍.